frontendIcons/src/main/resources/discover_dark.svg

```diff
@@ -18,7 +42,7 @@
         <dc:format>image/svg+xml</dc:format>
         <dc:type
            rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
-        <dc:title></dc:title>
+        <dc:title />
       </cc:Work>
```

Comment:
Whoops... weird

Reply:
But there doesn't seem to be any syntax error... maybe try 1) joining `<cc:Work` and the next line and/or 2) pushing the file through some XML verifier for sure

Reply:
It was just generated by Inkscape and it works, so I wouldn't worry about it